static/js/components/dashboard/CourseAction.js

```diff
@@ -87,11 +90,15 @@ export default class CourseAction extends React.Component {
     }
   }
 
+  isFutureCourse = (run: CourseRun): boolean =>
+    R.not(isEnrollableRun(run)) || _.isEmpty(run.course_id)
```

Comment:
why do you need to check `_.isEmpty(run.course_id)` here?

Reply:
it is in the issue statement https://github.com/mitodl/micromasters/issues/3688.

Reply:
Ah, ok. I think in that case the best thing would be probably to change the `isEnrollableRun` to check that, instead of making this new function